mod_reforged/hooks/skills/actives/reload_bolt.nut

```diff
@@ -10,7 +10,8 @@
 	o.onUse = function( _user, _targetTile	)
 	{
 		this.consumeAmmo();
-		this.getItem().setLoaded(true);
+		local ammoItem = this.getContainer().getActor().getItems().getItemAtSlot(this.Const.ItemSlot.Ammo);
+		this.getItem().setLoaded(true, ammoItem.getID(), ammoItem.getAmmoCost());
```

Comment:
I see here that ammo cost is somehow a function of ammo Item. This feels backwards to me. A weapon should define how much of ammo it consumes. What are your thoughts on this?